vaccine/models.py

```diff
@@ -0,0 +1,45 @@
+import uuid
+from django.db import models
+from django.utils import timezone
+from django_prometheus.models import ExportModelOperationsMixin
+from userprofile.validators import za_phone_number
+
+
+class VaccineRegistration(
+    ExportModelOperationsMixin("vaccineregistration"), models.Model
+):
+    GENDER_MALE = "male"
+    GENDER_FEMALE = "female"
+    GENDER_OTHER = "other"
```

Comment:
I think this is now `Male`, `Female`, and `Other`, because that's what EVDS accepts